Visual Studio 2022 Support

Swon404 opened the pull request:
Added Support for Visual Studio 2022, works with RC3

MidasTools committed:
Visual Studio 2022 Support

MidasTools committed:
update version to 1.23

pgerell commented:
Great.
I managed to build this once I removed the following elements from the project file
``` xml
<CopyVsixExtensionFiles>True</CopyVsixExtensionFiles>
<CopyVsixExtensionLocation>a:\</CopyVsixExtensionLocation>

```

MidasTools committed:
Removed CopyVsixExtensionFiles option

MidasTools committed:
Target correct versions for 2022

MidasTools committed:
Added fix from scigor:
Fixed uuid visualization by rewriting the the uuid visualization part…
… without the use of the custom dll

MidasTools committed:
Updated project settings

MidasTools committed:
Added VSIX ;)

MidasTools committed:
Merge branch '2022Support' of https://github.com/StewartWon/CPPDebuggerVisualizers into 2022Support

pps83 commented:
@KindDragon what about vs 2019/2022 support? Can you please update & publish them?
